### PR TITLE
docs: Update the download and install section of the OSM Manual Demo document

### PR DESCRIPTION
**Pull request by draychev**
This PR updates the **Download and install** section of the "OSM Manual Demo" document.

Goals:
  - simplify and clarify the verbiage
  - point to v0.8.2 release be explicit about the version of OSM this is relevant for (v0.8.2)
  - include instructions on downloading it on GNU/Linux, macOS, and Windows

---

This is one of a few steps to update the entire doc as part of #2845

Related PRs:
  - #3010

---

Once the entire doc is rewritten - all changes will be applied to the `release-v0.8` branch.

### CODE REVIEW

File: docs/content/docs/install/manual_demo/_index.md [hunk 1]
```diff
@@ -20,9 +20,29 @@ The OSM Manual Install Demo Guide is a step by step set of instructions to quick
...
+curl -L https://github.com/openservicemesh/osm/releases/download/${release}/osm-${release}-${system}-amd64.tar.gz | tar -vxzf -
+./${system}-amd64/osm version
+```
+
+Download the Windows OSM v0.8.2 binary via Powershell:
+```powershell
+wget  https://github.com/openservicemesh/osm/releases/download/v0.8.2/osm-v0.8.2-windows-amd64.zip -o osm.zip
+unzip osm.zip
+.\windows-amd64\osm.exe version
+```
```
Comment (shashankram) [inline docs/content/docs/install/manual_demo/_index.md:42]: This needs to go into OS specific sections, else it seems as though both need to be downloaded regardless. Could be a bit confusing if one doesn't have context.

I would reword this to something along the lines of:
```
For Linux and Mac .... :
...

For Windows ... :
...
```

**Review by SanyaKochhar — APPROVED** [review]

File: docs/content/docs/install/manual_demo/_index.md [hunk 2]
```diff
@@ -20,9 +20,29 @@ The OSM Manual Install Demo Guide is a step by step set of instructions to quick
...
 
-## Build or Download the OSM CLI
 
-Use the [installation guide](../../install) to install the `osm` cli.
+## Download and install the OSM command-line tool
+
+The `osm` command-line tool contains everything needed to install and configure Open Service Mesh.
+The binary is available on the [OSM GitHub releases page](https://github.com/openservicemesh/osm/releases/).
+
+Download the 64-bit GNU/Linux or macOS binary of OSM v0.8.2:
```
Comment (SanyaKochhar) [inline docs/content/docs/install/manual_demo/_index.md:29]: are we going to bump this version/doc with each release?

File: docs/content/docs/install/manual_demo/_index.md (hunk 1 above)
Comment (draychev) [reply docs/content/docs/install/manual_demo/_index.md:42]: Thank you!
It does indeed read like you need both!
Let me take your suggestion and add small subtitles.

File: docs/content/docs/install/manual_demo/_index.md (hunk 2 above)
Comment (draychev) [reply docs/content/docs/install/manual_demo/_index.md:31]: That would be the goal.
This is a bit of pain when we switch to versioned docs, but that pain on our side will be a seamless experience for the folks reading these docs and using OSM.
These should also be reflected here:
image


@SanyaKochhar how do you think about it?
We could perhaps not include patch and keep these at `v0.8`?

**Review by michelleN — APPROVED** [review]
